server/seed/seed.js

```diff
@@ -97,44 +101,39 @@ import {
     currentLeaderBoardLevel: 1,
     firstName: "User",
     lastName: "2",
+    city: "Orlando",
+    state: "FL",
   });
 
   const missionType1 = await Mission_Types.create({
-    id: 1,
```

Comment:
I need to have the mission types have an ID because they will be "set in stone" and not changed. As well as the explore levels. The activities are always being added on so they don't matter as much. 

But I'm not hard and fast on this. It was just easier to assign missions, users, etc knowing the ids for these.